### DISCUSSION TITLE
blog/replace-disqus-with-a-better-alternative

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#35] [Announcements]
# blog/replace-disqus-with-a-better-alternative

Disqus started polluting free users with unwanted ads; it’s time to replace Disqus with better open-source alternatives.

https://www.cesarsotovalero.net/blog/replace-disqus-with-a-better-alternative.html

**Comment by hewcaw** [comment]
Gotta check it right here to make sure it works :) Great article, thank you!

**Reply by cesarsotovalero** [reply]
It seems that it works 🙂 !

**Comment by freejacklee** [comment]
Me too.

**Comment by THENIROCK** [comment]
nice

**Reply by THENIROCK** [reply]
testing reply

**Comment by riojano0** [comment]
Only work with github pages?
Is a way to ensure that the repo-id and category-id are not used on another webpage to avoid flood or some wrong use?

**Reply by riojano0** [reply]
Ahhh, maybe configure the origins using the giscus.json =0
https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md

**Comment by Dcfontcom1** [comment]
Testing comment with https://dcfont.com

**Comment by GooglebellenNederland** [comment]
I am going to use it on my webiste https://bellenmerken.com/google/

**Comment by qianshancun** [comment]
Testing

**Comment by myServiceAccount** [comment]
<a href="#"><img src="https://camo.githubusercontent.com/546b4aa458ac9eeeaa55acb791a8d776692b5840ff49fe112e95900bee4afe39/68747470733a2f2f6b6f6d617265762e636f6d2f67687076632f3f757365726e616d653d53616d69725061756c31"  width="1" height="1"></a>

**Comment by myServiceAccount** [comment]
Testing...... 
<a href="#"><img src="https://camo.githubusercontent.com/546b4aa458ac9eeeaa55acb791a8d776692b5840ff49fe112e95900bee4afe39/68747470733a2f2f6b6f6d617265762e636f6d2f67687076632f3f757365726e616d653d53616d69725061756c31"  width="1" height="1"></a>

**Comment by myServiceAccount** [comment]
<a href="#"><img src="https://camo.githubusercontent.com/546b4aa458ac9eeeaa55acb791a8d776692b5840ff49fe112e95900bee4afe39/68747470733a2f2f6b6f6d617265762e636f6d2f67687076632f3f757365726e616d653d53616d69725061756c31"  width="1" height="1"></a>

**Comment by nailora-tr** [comment]
thank you https://www.nailora.com.tr

**Comment by zen0bit** [comment]
Work as expected 👍
https://osowoso.org
